### ISSUE TITLE
Carousal Add-on?

### ISSUE THREAD

**Issue by ddr5292** [billerickson/display-posts-shortcode#250]
Most of the plugins out there do not allow customization (or they are very heavy.) Adding this functionality to Display Posts would really be helpful.

**Comment by billerickson** [comment]
I can definitely see how that would be useful, but building a carousel addon would introduce opinionated design elements that might not work for everyone. Every user has unique design preferences, which is why Display Posts doesn't include any styles - it's built for users to style and customize themselves.

I've written a tutorial on [using template parts with Display Posts](https://www.billerickson.net/template-parts-with-display-posts-shortcode/), which is one way to do this.

Would it help if I wrote a separate tutorial showing how to customize Display Posts to use [Slick Slider](https://github.com/kenwheeler/slick) for the output?

The short version is:
- Add whatever parameters you would like to control in the shortcode. Ex: `[display-posts slick-slider="true" slick-dots="true" slick-arrows="false"]`
- Filter the opening markup ( `display_posts_shortcode_wrapper_open`) to add the necessary scripts and styles if the shortcode has `slick-slider="true"` and to customize the markup of the opening container
- Filter the individual item markup ( `display_posts_shortcode_output`) to output the appropriate markup for the slide.
- Filter the closing markup ( `display_posts_shortcode_wrapper_close` ) to customize the markup of the closing container.

**Comment by ddr5292** [comment]
Any help here would be appreciated. I believe it would be a popular post.
